src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ButtonComponent.tsx

```diff
@@ -85,11 +86,11 @@ export function ButtonComponent(props: IButtonProvidedProps) {
     const {
       org, app, instanceId,
     } = window as Window as IAltinnWindow;
-    FormDataActions.submitFormData(
-      `${window.location.origin}/${org}/${app}/api/${instanceId}`,
-      'Complete',
-      !ignoreWarnings,
-    );
+    dispatch(FormDataActions.submitFormData({
+      url: `${window.location.origin}/${org}/${app}/api/${instanceId}`,
```

Comment:
Would love to see this URL moved out to a urlHelper which the saga is in turn responsible for calling.

Reply:
Already done, the url parameter was not being used in the saga at all 🙄 Removed it from the method call.